File: Codeforces/C34B.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n, m;
    cin >> n >> m;
    vector<int> price(n);
    for (int i = 0; i < n; i++)
    {
        cin >> price[i];
    }

    sort(price.begin(), price.end());
    int earn = 0, index = 0, count = 0;
    while (index < n && price[index] < 0 && count < m)
    {
        earn -= price[index];
        index++;
        count++;
    }
    cout << earn;
}

File: Codeforces/C34B.py
n, m = map(int, input().split())
price = list(map(int, input().split()))
price.sort()
earn = 0
count = 0
for i in range(n):
	if(price[i] < 0 and count < m):
		earn += abs(price[i])
		count += 1
print(earn)
